tests/integration/test_dbapi_integration.py

```diff
@@ -1782,6 +1783,43 @@ def test_rowcount_insert(trino_connection):
         assert cur.rowcount == 1
 
 
+@pytest.mark.parametrize(
+    "legacy_prepared_statements",
+    [
+        True,
+        pytest.param(False, marks=pytest.mark.skipif(
+            trino_version() <= '417',
+            reason="EXECUTE IMMEDIATE was introduced in version 418")),
+        None
+    ]
+)
+def test_prepared_statement_capability_autodetection(legacy_prepared_statements, run_trino):
+    # start with an empty cache
+    trino.dbapi.must_use_legacy_prepared_statements = TimeBoundLRUCache(1024, 3600)
+    user_name = f"user_{t.monotonic_ns()}"
```

Comment:
(time is already imported)
```suggestion
    user_name = f"user_{time.monotonic_ns()}"
```

Reply:
I need the `time` package here.  The other import is the time `datetime.time` class.